<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PeakMan</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icons/resources/logo.ico</normaloff>:/icons/resources/logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_peakdet">
       <attribute name="title">
        <string>Peak Detection</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="detectPeaksButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Detect Peaks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="localThresholdGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Local Threshold</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QSpinBox" name="localThresholdSpinBox">
               <property name="suffix">
                <string> mV</string>
               </property>
               <property name="maximum">
                <number>65536</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>200</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="globalThresholdGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="title">
             <string>Global Threshold</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QSpinBox" name="globalThresholdSpinBox">
               <property name="suffix">
                <string> mV</string>
               </property>
               <property name="maximum">
                <number>65536</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>500</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="showGlobalThresholdCheckBox">
               <property name="text">
                <string>Display global threshold</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="minRRIntervalGroupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>115</width>
              <height>50</height>
             </size>
            </property>
            <property name="title">
             <string>Minimal RR Interval</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QSpinBox" name="minRRIntervallSpinBox">
               <property name="suffix">
                <string> ms</string>
               </property>
               <property name="maximum">
                <number>65536</number>
               </property>
               <property name="value">
                <number>270</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QScrollBar" name="horizontalScrollBar">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="ECGPlot" name="ecgPlot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="verticalSlider">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_artifact_processing">
       <attribute name="title">
        <string>Artifact Processing</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="updateIbiButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Update Interbeat
Intervals</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="jumpToSelectionButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Jump to
Selection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Manual Processing</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="artifactDetectionPushButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>95</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>Detect Artifacts</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>10</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="correctionMethodsLabel">
               <property name="text">
                <string>Correction Methods:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="linearInterpolationButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Linear Interpolation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="insertMissingPeaksButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Insert Missing Peaks</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QScrollBar" name="horizontalScrollBar_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="HistPlot" name="histPlot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="IBIPlot" name="ibiPlot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1059</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuOpenFile"/>
    <addaction name="menuCloseCurrentFile"/>
    <addaction name="separator"/>
    <addaction name="menuSavePeakPositions"/>
    <addaction name="menuSaveInterbeatIntervals"/>
    <addaction name="separator"/>
    <addaction name="menuQuit"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuInstructions"/>
    <addaction name="menuAboutPeakMan"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuAboutPeakMan">
   <property name="text">
    <string>About PeakMan</string>
   </property>
  </action>
  <action name="menuQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="menuInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="menuCloseCurrentFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close current file</string>
   </property>
  </action>
  <action name="menuResetView">
   <property name="text">
    <string>Reset View</string>
   </property>
  </action>
  <action name="menuOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="menuSavePeakPositions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Peak Positions</string>
   </property>
  </action>
  <action name="menuSaveInterbeatIntervals">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Interbeat Intervals</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ECGPlot</class>
   <extends>QWidget</extends>
   <header>ecgplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IBIPlot</class>
   <extends>QWidget</extends>
   <header>ibiplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HistPlot</class>
   <extends>QWidget</extends>
   <header>histplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
